### PR TITLE
frontend displays new hosts, waiting for backend

### PR DESCRIPTION
**Pull request by skupien**

### CODE REVIEW

**Comment by jakubsolecki** [comment]
@kraleppa please review it quickly because we need to merge changes scattered across branches ASAP

**Comment by kraleppa** [comment]
I'm playing bridge >:(

File: frontend/src/common/theme.ts [hunk 1]
```diff
@@ -28,8 +28,11 @@ const theme = createMuiTheme({
       "@global": {
         body: {
           background:
-            "linear-gradient(40deg, rgba(26,19,59,1) 0%, rgba(8,1,9,1) 20%, rgba(8,1,9,1) 80%, rgba(62,45,9,1) 100%);",
-          // "linear-gradient(40deg, #fe6b8b 30%, #ff8e53 90%)",
+            // "linear-gradient(40deg, rgba(26,19,59,1) 0%, rgba(8,1,9,1) 20%, rgba(8,1,9,1) 80%, rgba(62,45,9,1) 100%);",
+            // "linear-gradient(40deg, #050609 0%, #FA8334 30%, #D65780 90%, #050609 100%)",
+            "linear-gradient(40deg, #200122, #6f0000)",
+          // "linear-gradient(40deg, #403a3e, #be5869)",
+          // "repeating-radial-gradient(#FA8334, #D65780, #FA8334 2000px)",
```
Comment (kraleppa) [inline frontend/src/common/theme.ts:35]: Nice @skupien

**Review by kraleppa — CHANGES_REQUESTED** [review]
Something wrong with the alerts

File: backend/src/main/kotlin/pl/edu/agh/dockerbuddy/influxdb/InfluxDbProxy.kt [hunk 2]
```diff
@@ -101,7 +101,7 @@ class InfluxDbProxy {
                 + " |> range(start: $start, stop: $end)"
                 + " |> filter(fn: (r) => (" +
                     "r._measurement == \"host_stats\" and " +
-                    "r.host_id == \"$hostId.\" and " +
+                    "r.host_id == \"$hostId\" and " +
```
Comment (kraleppa) [inline backend/src/main/kotlin/pl/edu/agh/dockerbuddy/influxdb/InfluxDbProxy.kt:104]: FIX ALERTS >:(

File: frontend/src/common/theme.ts (hunk 1 above)
Comment (skupien) [reply frontend/src/common/theme.ts:35]: Thank you @kraleppa

File: backend/src/main/kotlin/pl/edu/agh/dockerbuddy/influxdb/InfluxDbProxy.kt (hunk 2 above)
Comment (skupien) [reply backend/src/main/kotlin/pl/edu/agh/dockerbuddy/influxdb/InfluxDbProxy.kt:104]: alerts are fixed on another branch. No need to fix them on right here right now

**Review by kraleppa — APPROVED** [review]
I haven't changed my mind, but @skupien said that he wants to merge

File: backend/src/main/kotlin/pl/edu/agh/dockerbuddy/influxdb/InfluxDbProxy.kt (hunk 2 above)
Comment (kraleppa) [reply backend/src/main/kotlin/pl/edu/agh/dockerbuddy/influxdb/InfluxDbProxy.kt:104]: i'll fix your face

**Pull request merged**